docs/literate/src/files/DGMulti_solver_mesh.jl

```diff
@@ -0,0 +1,220 @@
+#src # `DGMulti` solver and mesh
+
+# The basic idea and implementation of the solver [`DGMulti`](@ref) is already explained in
+# section ["Meshes"](@ref DGMulti).
+# Here, we want to give some examples and a quick overview about the options with `DGMulti`.
+
+# We start with a simple example we already used in the [tutorial about flux differencing](@ref fluxDiffExample).
+# There, we implemented a simulation with [`initial_condition_weak_blast_wave`](@ref) for the
+# 2D compressible Euler equations [`CompressibleEulerEquations2D`](@ref) and used the DG formulation
+# with flux differencing using volume flux [`flux_ranocha`](@ref) and surface flux [`flux_lax_friedrichs`](@ref).
+
+# Here, we want to implement the equivalent example, only now using the solver `DGMulti`
+# instead of [`DGSEM`](@ref).
+
+using Trixi, OrdinaryDiffEq
+
+equations = CompressibleEulerEquations2D(1.4)
+
+initial_condition = initial_condition_weak_blast_wave
+
+# To use the Gauss-Lobatto nodes again, we choose `approximation_type=SBP()` in the solver.
+# Since we want to start with a Cartesian domain discretized with squares, we use the element
+# type `Quad()`.
+dg = DGMulti(polydeg = 3,
+             element_type = Quad(),
+             approximation_type = SBP(),
+             surface_flux = flux_lax_friedrichs,
+             volume_integral = VolumeIntegralFluxDifferencing(flux_ranocha))
+
+mesh = DGMultiMesh(dg,
+                   cells_per_dimension=(32, 32), # initial_refinement_level = 5
+                   coordinates_min=(-2.0, -2.0),
+                   coordinates_max=( 2.0,  2.0),
+                   periodicity=true)
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, dg,
+                                    boundary_conditions=boundary_condition_periodic)
+tspan = (0.0, 0.4)
+ode = semidiscretize(semi, tspan)
+
+alive_callback = AliveCallback(alive_interval=10)
+analysis_callback = AnalysisCallback(semi, interval=100, uEltype=real(dg))
+callbacks = CallbackSet(analysis_callback, alive_callback);
+
+# Run the simulation with the same time integration algorithm as before.
+sol = solve(ode, RDPK3SpFSAL49(), abstol=1.0e-6, reltol=1.0e-6,
+            callback=callbacks, save_everystep=false);
+#-
+using Plots
+pd = PlotData2D(sol)
+plot(pd)
+#-
+plot(pd["rho"])
+plot!(getmesh(pd))
+
+# This simulation is not as fast as the equivalent with `TreeMesh` since the functions are not
+# implemented that efficient. On the other hand, the functions are more general and thus we
```

Comment:
```suggestion
# implemented that efficiently. On the other hand, the functions are more general and thus we
```
Might also be a good place to reference Figure 4 in the efficiency paper https://arxiv.org/abs/2112.10517 for a comparison between the different mesh types.